app/models/refinery/jobs/job.rb

```diff
@@ -36,7 +31,16 @@ def friendly_id_source
         title
       end
 
+      def self.latest(number = 5)
+        limit(number).order('created_at DESC')
+      end
+
+      def live?
+        !draft && published_at <= DateTime.now
```

Comment:
Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.